crates/pallet-history-seeding/src/lib.rs

```diff
@@ -40,14 +34,7 @@ pub mod pallet {
         /// Seed history with a remark
         #[pallet::call_index(0)]
         #[pallet::weight((T::WeightInfo::seed_history(remark.len() as u32), Pays::No))]
-        pub fn seed_history(origin: OriginFor<T>, remark: Vec<u8>) -> DispatchResult {
-            let who = ensure_signed(origin.clone())?;
-
-            ensure!(
-                Some(who.clone()) == Self::history_seeder(),
```

Comment:
not sure if I should still leave the validation here?

Reply:
No, I believe it is pointless